ml-agents/mlagents/trainers/sac/optimizer_torch.py

```diff
@@ -199,13 +199,9 @@ def __init__(self, policy: TorchPolicy, trainer_params: TrainerSettings):
             1e-10,
             self.trainer_settings.max_steps,
         )
-        self.policy_optimizer = torch.optim.Adam(
-            policy_params, lr=hyperparameters.learning_rate
-        )
-        self.value_optimizer = torch.optim.Adam(
-            value_params, lr=hyperparameters.learning_rate
-        )
-        self.entropy_optimizer = torch.optim.Adam(
+        self.policy_optimizer = torch.optim.Adam(policy_params)
+        self.value_optimizer = torch.optim.Adam(value_params)
+        self.entropy_optimizer = torch.optim.SGD(
```

Comment:
SGD makes the entropy_coef converge a lot faster than when using Adam. Since there is only one parameter to optimize, SGD is enough.